spec/support/components.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Support
+  module ComponentsTests
+    extend RSpec::Matchers::DSL
+
+    define :render do
+      match(&:render?)
+    end
```

Comment:
@kicferk1 `render?` is part ov ViewComponent library: https://viewcomponent.org/guide/conditional_rendering.html. And yes, this super-small matcher is just to enable single-line tests to avoid superfluous spec titles. Also `it { is_epxected.to render }` is just easier to understand (IMO).